salt/example.top

```diff
@@ -8,10 +8,10 @@ base:
         # we may be able to remove
         - elife.mercurial # elife-article-json
         - elife.aws-cli
-        - elife-libraries
-        - elife.jenkins-scripts
-        - elife.proofreader-php
         - elife.mysql57
         - elife.postgresql
+        - elife.jenkins-scripts
+        - elife.proofreader-php
```

Comment:
spooky that we need a particular order, but ok

Reply:
I'll send you the log and you can see for yourself!